MAINTAINERS.yml

```diff
@@ -193,9 +194,65 @@ Bluetooth:
     labels:
         - "area: Build System"
 
+"C library":
+    # Provisional entry, to be corrected
+    status: orphaned
+    collaborators:
+         - nashif
+         - andrewboie
+    files:
+         - lib/libc/
+    labels:
+         - "area: C Library"
+
+"CMSIS API layer":
+    # Provisional entry, to be corrected
+    status: orphaned
+    collaborators:
+         - nashif
+    files:
+         - lib/cmsis_rtos_v*/
+
+Debug:
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - nashif
+    files:
+        - include/debug/
+        - subsys/debug/
+        - tests/subsys/debug/
+    labels:
+        - "area: Debugging"
+
+DFU:
+    # Provisional entry, to be corrected
+    # TODO: spreadsheet not matching CODEOWNERS
+    status: maintained
+    maintainers:
+        - nvlsianpu
+    files:
+        - include/dfu/
+        - subsys/dfu/
+        - tests/subsys/dfu/
+
+"Device Tree":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - galak
+    collaborators:
+        - ulfalizer
+    files:
+        - scripts/dts/
+    labels:
+        - "area: Device Tree"
+
 "Display drivers":
     # Provisional entry, to be corrected
     status: orphaned
+    collaborators:
```

Comment:
status: maintained
maintainers:
 -vanwinkeljan
collaborators:
 -jfischer-phytec-iot

FIY: @vanwinkeljan

Reply:
Yep, we need @vanwinkeljan confirmation that he wants to be the maintainer :)

Reply:
> Yep, we need @vanwinkeljan confirmation that he wants to be the maintainer :)

Confirmed

Also add:
* include/display/
* include/drivers/display.h

@jfischer-phytec-iot wondering what to with `lib/gui/` and `subsys/fb/`?
I would add them under display for now.

Reply:
okay